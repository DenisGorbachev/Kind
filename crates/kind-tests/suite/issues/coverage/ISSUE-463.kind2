//The coverage check, Native type U60 --bug

type String {
    cons (x: U60) (xs: String)
    nil
}

Bits.from_hex (x: String) : U60
Bits.from_hex (String.cons '0' xs) = 2
Bits.from_hex (String.cons '1' xs) = 2