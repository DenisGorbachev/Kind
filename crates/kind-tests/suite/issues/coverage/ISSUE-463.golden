   WARN  This function does not cover all the possibilities!

      * Missing case : (Data.String.cons _ _) 

      /--[suite/issues/coverage/ISSUE-463.kind2:8:1]
      |
    7 |    
    8 |    Bits.from_hex (x: Data.String) : Data.U60
      |    v------------
      |    \Here!
    9 |    Bits.from_hex (Data.String.cons '0' xs) = 2


